discovery-provider/src/tasks/aggregates/__init__.py

```diff
@@ -20,13 +21,19 @@ def init_task_and_acquire_lock(
     db,
     redis,
     table_name,
-    aggregate_func,
+    aggregate_func: Callable[[Session, Redis], Any],
```

Comment:
Added a type for the callback, but couldn't get `mypy` to actually throw errors for it for some reason?!

Reply:
im ok with this if you are

Reply:
probably a bug in mypy, hopefully fixes itself in a new version or something?